webview/platforms/gtk.py

```diff
@@ -244,13 +244,18 @@ def close_window(self, *data):
         return False
 
     def on_drag_data(self, widget, drag_context, x, y, data, info, time):
-        if _dnd_state['num_listeners'] > 0 and data.get_text():
-            files = [
-                (os.path.basename(value), value.replace('file://', ''))
-                for value
-                in data.get_text().split('\n')
-                if value.startswith('file://')
-            ]
+        text = data.get_text()
+        if text and _dnd_state['num_listeners']:
+            if text.startswith('/'):
```

Comment:
This check won't do, as it will treat text starting with a slash as a file name. I cannot reproduce this on Ubuntu, but at very least it should maybe check that strings starting with a slash is a valid path name.

Reply:
Makes sense! I've updated the PR.

It's interesting that Fedora and Ubuntu behave differently here. :thinking: